src/utils/index.ts

```diff
@@ -0,0 +1,73 @@
+// 1. result Input에 따라서 계산하는 함수 (Main)
+export const caculateResultInput = (resultInput: string) => {
+    const operatorIdx = findOperatorIdx(resultInput);
+    const operator = resultInput[operatorIdx];
+    const operatorPrevNumber = extractOperatorPrevNumber(resultInput, operatorIdx);
+    const operatorNextNumber = extractOperatorNextNumber(resultInput, operatorIdx);
```

Comment:
연산자를 알고있으니 slice를 할 필요없이 split으로 해결해볼 수 있어보여요~
```suggestion
     const [operatorPrevNumber, operatorNextNumber] = resultInput
       .split(operator)
       .map(Number);
```